recipe/meta.yaml

```diff
@@ -1,58 +1,51 @@
-{% set cudnn_version= "8.2.1.32" %}
+{% set cudnn_version= "8.9.2.26" %}
 {% set build_number= "0" %}
-# The cuDNN installers can be downloaded from https://developer.nvidia.com/cudnn
-# The download requires authetication, for this recipe to work download the
-# packages to an staging location and adjust src_dir to this location.
+# The cuDNN installers can be downloaded from https://developer.nvidia.com/rdp/cudnn-download
+# The download requires authentication, for this recipe to work download the
```

Comment:
WOUld it be worth asking team Sirius to add the credentials into the worker's `.netrc` file since conda-build supports reading credentials from that?

Reply:
It's a website sign in then a manual download from a user account, which AFAIU a `.netrc` file doesn't handle? I found a resource [here](https://forums.developer.nvidia.com/t/download-cudnn-via-wget-or-curl/48952/11) which describes a way, but it involves putting an auth token into the recipe, so if we make the recipe private this might be ok.

Reply:
scratch that, the token expires. I started a chat on the sirius slack channel about this